src/unstract/sdk/adapters/x2text/llm_whisperer_v2/src/constants.py

```diff
@@ -0,0 +1,106 @@
+import os
+from enum import Enum
+
+
+class Modes(Enum):
+    NATIVE_TEXT = "native_text"
+    LOW_COST = "low_cost"
+    HIGH_QUALITY = "high_quality"
+    FORM = "form"
+
+
+class OutputModes(Enum):
+    LAYOUT_PRESERVING = "layout_preserving"
+    TEXT = "text"
+
+
+class HTTPMethod(Enum):
+    GET = "GET"
+    POST = "POST"
+
+
+class WhispererHeader:
+    UNSTRACT_KEY = "unstract-key"
+
+
+class WhispererEndpoint:
+    """Endpoints available at LLMWhisperer service."""
+
+    TEST_CONNECTION = "test-connection"
+    WHISPER = "whisper"
+    STATUS = "whisper-status"
+    RETRIEVE = "whisper-retrieve"
+
+
+class WhispererEnv:
+    """Env variables for LLM whisperer.
+
+    Can be used to alter behaviour at runtime.
+
+    Attributes:
+        POLL_INTERVAL: Time in seconds to wait before polling
+            LLMWhisperer's status API. Defaults to 30s
+        MAX_POLLS: Total number of times to poll the status API.
+            Set to -1 to poll indefinitely. Defaults to -1
+    """
+
+    POLL_INTERVAL = "ADAPTER_LLMW_POLL_INTERVAL"
+    MAX_POLLS = "ADAPTER_LLMW_MAX_POLLS"
+
+
+class WhispererConfig:
+    """Dictionary keys used to configure LLMWhisperer service."""
+
+    URL = "url"
+    PROCESSING_MODE = "mode"
+    OUTPUT_MODE = "output_mode"
+    UNSTRACT_KEY = "unstract_key"
+    MEDIAN_FILTER_SIZE = "median_filter_size"
+    GAUSSIAN_BLUR_RADIUS = "gaussian_blur_radius"
+    LINE_SPLITTER_TOLERANCE = "line_splitter_tolerance"
+    LINE_SPLITTER_STRATEGY = "line_splitter_strategy"
+    HORIZONTAL_STRETCH_FACTOR = "horizontal_stretch_factor"
+    PAGES_TO_EXTRACT = "pages_to_extract"
+    MARK_VERTICAL_LINES = "mark_vertical_lines"
+    MARK_HORIZONTAL_LINES = "mark_horizontal_lines"
+    PAGE_SEPARATOR = "page_seperator"
+    URL_IN_POST = "url_in_post"
+    LANG = "lang"
+    TAG = "tag"
+    FILE_NAME = "file_name"
+    USE_WEBHOOK = "use_webhook"
+    WEBHOOK_METADATA = "webhook_metadata"
+    TEXT_ONLY = "text_only"
+
+
+class WhisperStatus:
+    """Values returned / used by /whisper-status endpoint."""
+
+    PROCESSING = "processing"
+    PROCESSED = "processed"
+    DELIVERED = "delivered"
+    UNKNOWN = "unknown"
+    # Used for async processing
+    WHISPER_HASH = "whisper_hash"
+    STATUS = "status"
+
+
+class WhispererDefaults:
+    """Defaults meant for LLM whisperer."""
+
+    MEDIAN_FILTER_SIZE = 0
```

Comment:
@harini-venkataraman defaults for mode and output mode?

Reply:
@johnyrahul Have added the defaults  in the method - `get_whisperer_params`